google/cloud/opentelemetry/internal/recordable.cc

```diff
@@ -301,10 +301,9 @@ void Recordable::SetDuration(std::chrono::nanoseconds duration) noexcept {
 void Recordable::SetInstrumentationScope(
     opentelemetry::sdk::instrumentationscope::InstrumentationScope const&
         instrumentation_scope) noexcept {
-  SetAttribute("otel.instrumentation_library.name",
-               instrumentation_scope.GetName());
-  SetAttribute("otel.instrumentation_library.version",
-               instrumentation_scope.GetVersion());
+  SetAttribute("otel.scope.name", instrumentation_scope.GetName());
+  if (instrumentation_scope.GetVersion().empty()) return;
+  SetAttribute("otel.scope.version", instrumentation_scope.GetVersion());
```

Comment:
I'd suggest `if (!x.empty()) SetAttribute(a, x)` instead.  It's not like the emptiness of the version makes the rest of the function improper, even though the effect is currently the same because the rest of the function is currently empty.

Reply:
Done.